src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsKruxObject.java

```diff
@@ -95,7 +83,10 @@ public String getKxsegs() {
   }
 
   private String wrapSegs(String segmentsLocalStorage) {
-    String ksgmnt = "\"ksgmnt\":[";
+    if (segmentsLocalStorage.equals("")) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1132)